test/ad/distributions.jl

```diff
@@ -18,14 +18,18 @@
     to_posdef_diagonal(a::AbstractVector) = Diagonal(a.^2 .+ 1)
 
     # Define adjoints for Tracker
-    to_posdef(A::TrackedMatrix) = Tracker.track(to_posdef, A)
-    Tracker.@grad function to_posdef(A::TrackedMatrix)
-        data_A = Tracker.data(A)
-        S = data_A * data_A' + I
-        function pullback(∇)
-            return ((∇ + ∇') * data_A,)
+    if AD == "All" || AD == "ForwardDiff_Tracker"
```

Comment:
I guess we should split `ForwardDiff_Tracker` in `ForwardDiff` and `Tracker`

Reply:
Yes, I agree, but I wanted to ask first if there was a reason for Tracker/ForwardDiff being tested together in the existing code

Reply:
I guess the reason was mainly that FiniteDifferences wasn't used at all and everything was compared against ForwardDiff (at least that was the status a week ago when I started refactoring the tests IIRC).